airflow/contrib/hooks/spark_submit_hook.py

```diff
@@ -206,9 +207,10 @@ def _get_spark_binary_path(self):
         # the executing user
         if self._connection['spark_home']:
             connection_cmd = [os.path.join(self._connection['spark_home'], 'bin',
-                                           self._connection['spark_binary'])]
+                                           self._connection['spark_binary'] or
+                                           "spark-submit")]
         else:
-            connection_cmd = [self._connection['spark_binary']]
+            connection_cmd = [self._connection['spark_binary'] or "spark-submit"]
```

Comment:
```suggestion
            connection_cmd = [self._connection['spark_binary']]
```

This shouldn't be needed here as it's already set in _resolve_connection (lets keep the defaulting to one place)